modules/virt-creating-data-volumes-using-storage-api.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-storage-api_{context}"]
+
+= Creating data volumes using the Storage API
+
+When you create a data volume using the *storage* API, the CDI optimizes your PVC allocation based on the type of storage supported by your selected storage class. You only have to specify the data volume name, namespace, and the amount of storage that you want to allocate.
```

Comment:
This is a little tricky for PVC since it's the name of the API and also the object. Here's what I did: 

- In every case where I was referring to the ACTUAL PVC, I defined the acronym, except in the call-outs, where I only do it once. There are not that many places in the text that reference PVC but if you think that approach is too busy, let me know. I think it's better to err on the side of being explicit in this case, as it's confusing to name to API this way.
- CDI is obviously more straight-forward so I defined it once per module/assembly.